src/utils/cat.js

```diff
@@ -2,19 +2,19 @@ export default class Cat {
   constructor(name) {
     this.id = 0;
     this.name = name;
-    this._amountOfCheese = 0
+    this._amountOfCheese = 0;
   }
 
   get amountOfCheese() {
     return this._amountOfCheese;
   }
 
   set amountOfCheese(cats) {
-    this._amountOfCheese =  this._randomAmountOfCheese(cats);
+    this._amountOfCheese = this._randomAmountOfCheese(cats);
   }
 
   _randomAmountOfCheese(cats) {
-    return Math.floor(Math.random() * cats.length) + 1;
+    return Math.floor(Math.random() * 4) + 1;
```

Comment:
The cats.length is actually meant to use the number of cats that we have to generate a random number within that range.
Using a fixed value of "4" means that, if in the future we decided to increase the number of cats, we will have to change that number again.
Please look into it.

Overall, it's a great work and I like it.
Thanks.

Reply:
As discussed on Discord I will merge and look into the error that appeared when using cats.length later.